src/game/AI/ScriptDevAI/scripts/eastern_kingdoms/scarlet_monastery/boss_mograine_and_whitemane.cpp

```diff
@@ -22,6 +22,7 @@ SDCategory: Scarlet Monastery
 EndScriptData */
 
 #include "AI/ScriptDevAI/include/sc_common.h"
+#include "AI/ScriptDevAI/include/sc_instance.h"
```

Comment:
Sorry for all the additional includes, my IDE does that automatically and I don't always catch it.
sc_instance.h and SharedDefines.h are unnecessary